src/Datadog.Trace.ClrProfiler.Native/pal.h

```diff
@@ -58,7 +58,7 @@ inline WSTRING DatadogLogFilePath() {
   return ToWSTRING(program_data +
                    R"(\Datadog .NET Tracer\logs\dotnet-tracer-native.log)");
 #else
-  return "/var/log/datadog/dotnet/dotnet-tracer-native.log"_W;
+  return WStr("/var/log/datadog/dotnet/dotnet-tracer-native.log");
```

Comment:
bad path but it is already fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/66/files